packages/x-components/src/x-modules/facets/wiring.ts

```diff
@@ -4,9 +4,23 @@ import { createRawFilters } from '../../utils/filters';
 import { wireCommit, wireService, wireServiceWithoutPayload } from '../../wiring/wires.factory';
 import { filter, mapWire } from '../../wiring/wires.operators';
 import { createWiring } from '../../wiring/wiring.utils';
-import { XEventPayload } from '../../wiring/index';
+import { namespacedWireCommit, XEventPayload } from '../../wiring/index';
 import { DefaultFacetsService } from './service/facets.service';
 
+/**
+ * `facets` {@link XModuleName | XModule name}.
+ *
+ * @internal
+ */
+const moduleName = 'facets';
+
+/**
+ * WireCommit for {@link SearchXModule}.
+ *
+ * @internal
+ */
+const wireCommitModule = namespacedWireCommit(moduleName);
```

Comment:
The `wireCommit` that the wiring in the module uses should be replaced by a `namespacedWireCommitWithoutPayload`.

The wires that commit mutations with payload should keep using the one you created here.